src/drive/styles/toolbar.styl

```diff
@@ -92,7 +91,9 @@
 
     .fil-public-download
         display none
-
+    .fil-toolbar-menu
+        .fil-toolbar-menu-recipients
+            display inline-flex !important
```

Comment:
Why is it needed ?

Reply:
because I've a conflict with the previous media query (@media (max-width: 63.938rem)). Its style are loaded before the small-screen() and I've a display none at the end :/ 

I can play with the declaration order, I'm pretty just it will work but... 

Is there a way in UI to have a media query between max-width: 63 & min-width > small-screen?

Reply:
<img width="368" alt="Capture d’écran 2020-03-24 à 09 26 17" src="https://user-images.githubusercontent.com/1107936/77404120-8cf92a80-6db1-11ea-9f36-2ade7d719952.png">

Reply:
I think [tablet()](https://github.com/cozy/cozy-ui/blob/master/stylus/settings/breakpoints.styl#L193) is what you are looking for?

Reply:
medium-device() doest the trick

Reply:
tablet is exactly what I needed thanks!